src/components/value-list/value-list.tsx

```diff
@@ -457,6 +458,14 @@ export class ValueList<
     }
   };
 
+  @Listen("calciteValueListItemDragHandleBlur")
+  handleValueListItemBlur(event: CustomEvent<ListItemAndHandle>): void {
+    const { item, handle } = event.detail;
```

Comment:
[Per our conventions](https://github.com/Esri/calcite-components/blob/master/conventions/README.md#privateinternal-events), the propagation of custom events should be stopped after being handled.